packages/docs-hyper-link-ui/src/commands/operations/popup.operation.ts

```diff
@@ -81,3 +82,44 @@ export const ShowDocHyperLinkEditPopupOperation: ICommand<IShowDocHyperLinkEditP
         return true;
     },
 };
+
+export interface IShowDocHyperLinkInfoPopupOperationParams {
+    linkId: string;
+    segmentId?: string;
+    unitId: string;
+    rangeIndex: number;
+    segmentPage?: number;
+}
+
+export const ToggleDocHyperLinkInfoPopupOperation: ICommand<IShowDocHyperLinkInfoPopupOperationParams> = {
+    type: CommandType.OPERATION,
+    id: 'docs.operation.show-hyper-link-info-popup',
```

Comment:
According to the naming convention, use singular format:

```suggestion
    id: 'doc.operation.show-hyper-link-info-popup',
```